.github/workflows/arm64-build.yml

```diff
@@ -0,0 +1,138 @@
+on:
+  pull_request:
```

Comment:
Enabling it on pull requests first so that we run the job in this PR and I know I got things right. Will remove before merging.